connectivity/FEATURE_BLE/include/ble/gatt/GattCallbackParamTypes.h

```diff
@@ -384,6 +384,29 @@ struct GattHVXCallbackParams {
 
 };
 
+/**
+ * Gatt Data Sent Attribute related events
+ *
+ * Used by `onDataSent`
+ */
+struct GattDataSentCallbackParams {
+
+    /**
+     * The handle of the connection that triggered the event.
+     */
+    ble::connection_handle_t connHandle;
+
+    /**
+     * Attribute Handle to which the event applies
+     */
+    GattAttribute::Handle_t handle;
+
```

Comment:
Side note: As an improvement I'd like to forward the status as unsuccessful _send_ attempts are not forwarded to the application today. 

This doesn't have to be in to accept this PR.

Reply:
This will be possible in a future update since the parameter types can now be updated more easily. It should be backwards compatible as long as the existing parameter struct members aren't removed (only new ones added).